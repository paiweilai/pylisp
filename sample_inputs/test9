(times 5 (plus 4 2))
(quotient (quotient 15 3) 2)
(remainder 7 (remainder 3 4))
(less (plus 2 3) 6)
(greater 5 6)
(eq (quote a) (quote a))
(eq (quote a) (quote b))
(int (plus 4 5))
(int (cons 4 5))
(atom (quote (7 . 10)))
(atom 7)
(atom (car (quote (A . 10))))
(null (eq 2 (plus 1 +1)))
(null (quote ()))
(cons (car (quote (7 . 10))) (cdr (quote (7 . 10))))
