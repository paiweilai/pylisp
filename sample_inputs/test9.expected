30
2
1
T
NIL
T
NIL
T
NIL
NIL
T
T
NIL
T
(7 . 10)
